### ISSUE TITLE
Conferences are too expensive!

### ISSUE THREAD

**Issue by bojanxmk** [kitze/awesome-conference-practices#7]
Its true that developers are getting good paychecks these days but hear me out. 

#1 case:
Imagine being a newbie developer that just started learning React (its difficult to say Vue on Kitze's repo lol), and you want to at least connect with people that go on these conferences so you can be in touch with people that share similar interest. You have no job, or you are doing some open source project that brings you a small income that will only get you to pay your bills. How to actually afford 500€ ticket?

#2 case:
You are not living in well developed countries where your minimal salary is at least 4-5times higher that the conference ticket price. How can you purchase a ticket that is 500€ in a country that has average salary of 300€? You would say yeah, but developers usually earn more than the average salary, which is true but still how to extract 70% of your salary for the ticket only? Where is the accommodation, airplane ticket, your actual stay there for 2-3days?

I agree that you have to pay the speakers, organizers, the venue and all of that stuff but still, people that are limited with funds should be also available to visit these conferences. 

A real world example is that if i visit http://reactforbeginners.com by Wes Bos, i actually get a coupon that says "You are visiting from Serbia (which unfortunately belongs to those 'third world countries'), here is a 57% discount IF YOU NEED it. The key here is that its offered, and i know that (including myself) if the discount is not actually needed, and the buyer can afford the whole price he will not use the discount, because i think that is how our dev culture is, since we all know how is it to invest in something and not get paid (hint: Open Source).

This was something that was going through my head since Kitze suggested me to go to a conference and i started doing calculations how to accumulate those 500+€ for only a ticket! 

Conference organizers, please think of a solution :)

**Comment by kitze** [comment]
@bojanxmk good point. In most of the cases, employers are sponsoring their employees with a ticket. I think no one would pay 500 euro for a conference out of their pocket.

**Comment by csprance** [comment]
@kitze I disagree with that and I think a lot of smaller indie developers would as well.

Sometimes you don't work for a company that 
```
A) Can't shell out the cash to send employees
B) Wont shell out the cash to send employees
```

I have paid out of pocket multiple times and it sucks. Not to mention each year I feel like they get more and more expensive and start to try and break down the conference into more and more package tickets that you need in order to attend certain talks. 

You want the VR talk oh sorry that's an extra $400 even though you got the all access pass.

**Comment by MrSunshyne** [comment]
This is a reality for developers living on remote islands. Considering the plane ticket price to reach EU is 3-4 times higher than the conference ticket itself. US conferences are not even worth considering.

The solution is then to have local conferences by developers for developers. Then it flips, now its expensive to invite speakers from overseas. 

Thank god the internet exists.

**Comment by bojanxmk** [comment]
@MrSunshyne we basically live on Internet, it would be nice to socialize with people that share similar interests on these conferences. That should be the main focus of a conference. 

@csprance "Conferences should be about networking and sharing information not making as much money as possible." - well said, exactly on point!

**Comment by brentvatne** [comment]
When I was a student I volunteered at a conference that I really wanted to go to in order to be able to attend. In doing so I was able to meet more people and I still had time to watch the talks. As someone who has now also organized a conference I know that there are a good number of people who take this approach -- I received plenty of emails about it. Perhaps conferences could say upfront how many volunteers they need, what the positions would be roughly, and where to apply?

**Comment by MrSunshyne** [comment]
@bojanxmk Absolutely agree. But the main hurdle remain travelling costs. A plan ticket to anywhere in EU would cost $1000 upwards (from where I am). I'm also curious to know what the average travelling cost is for ppl already in EU.

Even @brentvatne's suggestion of volunteering cannot be considered, as I imagine no organiser would be willing to sponsor that much per volunteer.

**Comment by Taym95** [comment]
Hello everyone! I'm Junior frontend developer and conferences tickets are too expensive for me! in the last React Amsterdam conference if @satya164 didn't give me a ticket I wouldn't be able to attend this conference and I wouldn't be able to watch your(@kitze) talk live and meeting all react community!

**Comment by operatino** [comment]
At React Amsterdam we gave away around 20% tickets for free to volunteers, lightning talk speakers, community representatives/contest winners and to diversity scholarship programme members. There's a lot of opportunities getting free tickets, even by just joining local React communities and winning tickets there for a retweet or by asking diversity scholarship support.

Meetups are definitely good alternatives, where you can listen and meet more or less the same people from the conference. Free talk recordings is also a thing, plus just inviting people over for a beer/coffee around conference days also works.

Suggestions above does not help if you're living remotely, or don't have decent community in town, but then diversity scholarships + volunteering and lowcoster flight can do.
